### PR TITLE
put the guide into main deployment page

### PR DESCRIPTION
**Pull request by rauerhans**
## Summary

create the guide page from 

https://pluralsh.notion.site/Getting-Started-With-Plural-Fleet-Management-b7fa404b35244832bad98e06671799f0

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: pages/getting-started/deployments.md [hunk 1]
```diff
@@ -3,3 +3,35 @@ title: Getting Started with Plural CD
...
+
+The general process of setting up your instance is pretty straightforward, and we have documentation and supporting materials below to guide each step:
+
+- set up your management cluster which will host the plural console and supporting operators
+- connect your kubernetes clusters you’d like to manage to that management cluster
+- define services, pipelines, and global services to start shipping workloads into your cluster fleet
+
+## Intro Video
+
+We think the easiest way to understand the workflow is to take a look at our intro video; we have an upload of it [here](https://assets.plural.sh/plural-up.mp4) (It is about 14 minutes long).
```
Comment (michaeljguarino) [inline pages/getting-started/deployments.md:20]: I think we actually have a youtube video link now: https://www.youtube.com/watch?v=caDlvskRkAw&ab_channel=Plural

I'm not actually sure how to do video embeds though, there's probably an example elsewhere in the markdown code.

**Comment by rauerhans** [comment]
`react-embed` which we are using (https://github.com/pluralsh/design-system/blob/e45314b93c74708b654331077c1c6feb9a948e4c/src/markdoc/components/Embed.tsx) 
interprets all `mp4` files as audio, so embedding the video natively doesn't work in that format:
https://github.com/streamich/react-embed/issues/866

Maybe if we store the asset as .webm it'll display correctly

File: pages/getting-started/deployments.md (hunk 1 above)
Comment (rauerhans) [reply pages/getting-started/deployments.md:20]: it's all youtube videos in the examples, if we have a yt video of that now, that's perfect, that'll work

**Review by michaeljguarino — APPROVED** [review]

**Comment by samweaver** [comment]
approved, squashed and merged

**Pull request merged**